executor/aggfuncs/aggfuncs.go

```diff
@@ -108,6 +109,24 @@ type AggFunc interface {
 	// partial result and then calculates the final result and append that
 	// final result to the chunk provided.
 	AppendFinalResult2Chunk(sctx sessionctx.Context, pr PartialResult, chk *chunk.Chunk) error
+
+	// GetDefaultValue gets the default value when the aggregation function's input is null.
+	// According to MySQL, default values of the aggregation function are listed as follows:
+	// e.g.
+	// Table t which is empty:
+	// +-------+---------+---------+
+	// | Table | Field   | Type    |
+	// +-------+---------+---------+
+	// | t     | a       | int(11) |
+	// +-------+---------+---------+
+	//
+	// Query: `select a, avg(a), sum(a), count(a), bit_xor(a), bit_or(a), bit_and(a), max(a), min(a), group_concat(a) from t;`
+	// +------+--------+--------+----------+------------+-----------+----------------------+--------+--------+-----------------+
+	// | a    | avg(a) | sum(a) | count(a) | bit_xor(a) | bit_or(a) | bit_and(a)           | max(a) | min(a) | group_concat(a) |
+	// +------+--------+--------+----------+------------+-----------+----------------------+--------+--------+-----------------+
+	// | NULL |   NULL |   NULL |        0 |          0 |         0 | 18446744073709551615 |   NULL |   NULL | NULL            |
+	// +------+--------+--------+----------+------------+-----------+----------------------+--------+--------+-----------------+
+	GetDefaultValue() (v types.Datum)
```

Comment:
Maybe treat this as a trait of the `AggFuncDesc` is better? Like `EvalNullValueInOuterJoin`